src/functions/index.ts

```diff
@@ -149,8 +147,8 @@ export function main(initialString?: string, jsonSource?: string) {
 // console.log(main(`buy, bernard unfortunate, fries`));
 // console.log(main(`table, adam smith, fries`));
 // console.log(main(`buy, adam smith, fries\nbuy, adam smith, fries\nbuy, adam smith, fries`));
-console.log(main(`table, adam smith, fries\ntable, adam smith, fries\ntable, adam smith, fries\n Audit, Resources`));
-// console.log(main(`order, tuna, 5\norder, tuna, 5\norder, tuna, 5\norder, tuna, 5`));
+console.log(main(`table, adam smith, fries\ntable, adam smith, fries\n Audit, Resources \ntable, adam smith, fries`));
+// console.log(main(`order, tuna, 2, princess chicken, 5\norder, tuna, 5\norder, tuna, 5\norder, tuna, 5`));
 // console.log(main(`order, potatoes, 30\n order, water, 5`));
 // console.log(main(`table, barbara smith, bernard unfortunate, adam smith, tuna cake, fries, fries\n Audit, Resources\nThrow trash away\nOrder, chicken, 1\nthrow trash away`));
 // console.log(main(`table, barbara smith, bernard unfortunate, adam smith, tuna cake, fries, fries\n throw trash away`, `../../json/number.json`));
```

Comment:
Why do we keep such commented out code?
Is it maintained?
If there are some important use-cases, maybe they should be:
- commented (what for etc)
- extracted?
- presented as tests?

Reply:
Most of this cases have already test coverage, thou it's easier for me to be able to call them directly in this function